packages/mui-material-next/src/styles/Theme.types.ts

```diff
@@ -25,7 +25,7 @@ export interface MD3Palettes {
   primary: MD3Tones;
   secondary: MD3Tones;
   tertiary: MD3Tones;
-  neutral: MD3Tones;
+  neutral: MD3Tones & { 17: string; 22: string; 92: string };
```

Comment:
Could we extract these to a new local type so it's easier to figure out what they represent? Something along the lines of `MD3DefaultThemeNeutralTones`?

Reply:
Updated